src/components/SkyBrowser/SkybrowserFocusEntry.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../common/Input/Button/Button';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import InfoBoxSkybrowser from './InfoBoxSkybrowser';
+import TooltipSkybrowser from './TooltipSkybrowser';
+import styles from './SkybrowserFocusEntry.scss';
+
+//import { clamp } from 'lodash/number';
+class OpacitySlider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: 100
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+    // Ensure the image has an id
+    if(this.props.identifier) {
+      this.props.setOpacity(Number(this.props.identifier), Number(this.state.value/100));
+    }
+  }
+
+  render() {
+    return   <div className={styles.slidecontainer}><input type="range" min="0" max="100" value={this.state.value}  className={styles.slider} onChange={this.handleChange} /> </div>;
```

Comment:
please remove extra spaces before first `div` and add line breaks, for readabiliy :)